typescript/sdk/src/deploy/types.ts

```diff
@@ -27,6 +27,13 @@ export type OwnableConfig<Keys extends PropertyKey = PropertyKey> = z.infer<
   ownerOverrides?: Partial<Record<Keys, Address>>;
 };
 
+export function extractOwnerAddress(config: Owner): Address {
+  if (typeof config === 'string') {
+    return config;
+  }
+  return config.owner;
+}
```

Comment:
this is how I'm extracting an owner address from an `Owner` from an `IsmConfig` - trusting that the caller of the ISM module has done the ICA stuff if they need to on their end

Reply:
I removed this from the "runtime" config in my PR as well https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3820
imo we shouldnt pollute all of the deployers with ICA resolution code